modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -2599,29 +2602,85 @@ public void testDiagnosticPageLocksTracker() throws Exception {
 
         ignite.cluster().active(true);
 
-        String dir = U.defaultWorkDirectory() + "/diagnostic/";
+        File default0 = new File(U.defaultWorkDirectory() + "/" + DEFAULT_TARGET_FOLDER + "/");
+        File testDir = new File(U.defaultWorkDirectory() + "/diagnostic_test_dir/");
 
-        assertEquals(EXIT_CODE_OK, execute("--diagnostic"));
-        assertEquals(EXIT_CODE_OK, execute("--diagnostic", "help"));
-        assertEquals(EXIT_CODE_OK, execute("--diagnostic", "pageLocks", "help"));
+        if (default0.exists())
```

Comment:
You should move cleaning files/directories from test to `befoteTest()`/`afterTest()` methods